sdk/keyvault/keyvault-admin/src/lro/backup/operation.ts

```diff
@@ -61,7 +56,7 @@ export class KeyVaultBackupPollOperation extends KeyVaultAdminPollOperation<
    * Tracing the fullBackup operation
    */
   private fullBackup(options: FullBackupOptionalParams): Promise<FullBackupResponse> {
-    return withTrace("fullBackup", options, (updatedOptions) =>
+    return tracingClient.withSpan("KeyVaultBackupClient.fullBackup", options, (updatedOptions) =>
```

Comment:
I see that in general you are using the class name. Here it's a bit different, is that okay?

Reply:
Thanks, good catch - I fixed it